src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.OpenTextBufferManager.cs

```diff
@@ -323,12 +341,6 @@ internal void ApplyConflictResolutionEdits(IInlineRenameReplacementInfo conflict
                     // 1. Undo any previous edits and update the buffer to resulting document after conflict resolution
                     _session.UndoManager.UndoTemporaryEdits(_subjectBuffer, disconnect: false);
 
-                    var preMergeSolution = _session._baseSolution;
-                    var postMergeSolution = conflictResolution.NewSolution;
-
-                    var diffMergingSession = new LinkedFileDiffMergingSession(preMergeSolution, postMergeSolution, postMergeSolution.GetChanges(preMergeSolution), logSessionInfo: true);
-                    var mergeResult = diffMergingSession.MergeDiffsAsync(mergeConflictHandler: null, cancellationToken: cancellationToken).WaitAndGetResult(cancellationToken);
```

Comment:
:memo: In addition to moving to a background thread, this operation was lifted out of a loop over the open documents in `ApplyReplacements`.